app/lib/general.lib.js

```diff
@@ -152,6 +152,45 @@ function generateAbstractionPointDetail (pointDetail) {
   return abstractionPoint
 }
 
+/**
+ * Generate a string that represents an abstraction point based on the assumption the points have already been merged
+ *
+ * When abstracting water the point at which this is done can be described in several ways depending on the number of
+ * Nation Grid References are saved against the abstraction point. This function checks for these references and builds
+ * a string that defines the details of the abstraction point.
+ *
+ * This follows the same out put as generateAbstractionPointDetail but the points have already been merged
+ *
+ * @param {Object} pointDetail - Object containing all the details for the point
+ *
+ * @returns {String} a description of the abstraction point
+ */
+function generatePointDetail (pointDetail) {
```

Comment:
Might need to work on this name and docs.

My current understanding is limiting me to explain it better.